springfox-spring-integration/build.gradle

```diff
@@ -16,8 +16,10 @@ dependencies {
   compile project(':springfox-spi')
   compile project(':springfox-schema').sourceSets.main.output
   compile project(':springfox-spring-web')
+  compile project(':springfox-swagger-common')
   compile "com.athaydes.rawhttp:rawhttp-core:2.0"
   compileOnly "org.springframework.restdocs:spring-restdocs-mockmvc:2.0.3.RELEASE"
+  compileOnly "org.springframework.data:spring-data-rest-webmvc:$springDataRest"
```

Comment:
Can this be provided?

Reply:
Not sure to understand what you mean by "provided".